src/components/interaction-area/interaction-area.utils.js

```diff
@@ -0,0 +1,170 @@
+import { elementsCreator } from '../../utils/dom';
```

Comment:
Moving all interaction-areas function from 'player.util' and 'player.const' to component/interaction-areas  folder